src/main/java/com/learntechfree/ltf/entity/Article.java

```diff
@@ -0,0 +1,79 @@
+package com.learntechfree.ltf.entity;
+
+import lombok.*;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+import java.util.Set;
+
+@Entity
+@Table(name = "articles",uniqueConstraints = {
+        @UniqueConstraint(columnNames = {"title","slug"})
+})
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class Article extends AuditModel {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+
+    @Column(nullable = false)
+    private String title;
+
+    @Column
+    private String subTitle;
+
+    @Column(nullable = false)
+    private String slug;
+
+    @Column(nullable = false, columnDefinition = "TEXT")
+    private String body;
+
+    @Column
+    private String status;
+
+    @Column
+    private String tags;
+
+    @Column
+    private String featureImageUrl;
+
+    @Column(updatable = false, nullable = false)
+    private Long createdBy;
+
+    @Column
+    private Long updatedBy;
+
+    @Column
+    private Long publishedBy;
+
+    @Column(nullable = false)
+    private String userId;
+
+    @Column
+    private Timestamp publishedAt;
+
+    @Column
+    private Timestamp scheduledAt;
+
+    /**
+     * @nullable true for blog article
+     */
```

Comment:
Either add docs for all or remove for all, this makes code dirty.

Reply:
👍